CMakeLists.txt

```diff
@@ -43,3 +43,4 @@ add_subdirectory(utest-imgaos)
 add_subdirectory(utest-imgsoa)
 add_subdirectory(ftest-aos)
 add_subdirectory(ftest-soa)
+add_executable(new_target imgaos/aosinfrequentcolor.cpp)
```

Comment:
not sure what this is. I don't think it should be here tho.

Reply:
done